src/sqlParser/create.table.ts

```diff
@@ -74,6 +74,8 @@ export function createTable(tokens: Token[]): CreateTable {
       }
 
       continue;
+    } else if (isUnknown(token)) {
```

Comment:
I think we should merge this part with the conditional statement above.

`isString(token) && !ast.name` => `(isString(token) || isUnknown(token)) && !ast.name`